### PR TITLE
FT: set file backend as the default backend

### PR DESCRIPTION
**Pull request by LaurenSpiegel**
Merging this commit (which is already in master) will prevent only one worker from being created when S3BACKEND=scality

### CODE REVIEW

**Comment by LaurenSpiegel** [comment]
@ironman-machine thoughts?

File: lib/server.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ class S3Server {
 
 export default function main() {
     let clusters = _config.clusters || 1;
-    if (process.env.S3BACKEND) {
+    if (process.env.S3BACKEND === 'mem') {
```
Comment (DavidPineauScality) [inline lib/server.js]: wouldn't that line be enough for rel/1.1, rather than bringing the whole default commit ?

Comment (DavidPineauScality) [inline lib/server.js]: Note: As much as possible, I'd like to only bring the real fix required, rather than the full commit, if avoidable.

Comment (LaurenSpiegel) [inline lib/server.js]: Yes, we could do just that one line.

**Pull request closed without merging**